test/functional/tests/security/fuzzy/config/cache_line_size.yml

```diff
@@ -0,0 +1,15 @@
+- name: String
+  attributes:
+    name: Cls
+    value: '4'
+    size: '6'
```

Comment:
What does size mean? Number of characters?

Reply:
Yes